libheif/0001-Edit-CMakeLists.patch

```diff
@@ -0,0 +1,79 @@
+From 9d98552f1c053e9b3f50ceab09f05a8664333fae Mon Sep 17 00:00:00 2001
+From: L4cache <42721712+L4cache@users.noreply.github.com>
+Date: Sat, 7 Dec 2024 08:54:41 +0800
+Subject: [PATCH] Edit CMakeLists
+
+This patch updates VVdeC version and bypasses some problems during build process and not configurable.
+---
+ CMakeLists.txt        | 27 ++-------------------------
+ heifio/CMakeLists.txt |  2 +-
+ 2 files changed, 3 insertions(+), 26 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index c26ea684..f4c56a81 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -160,7 +160,7 @@ plugin_option(VVDEC "vvdec VVC decoder (experimental)" OFF OFF)
+ if (WITH_VVDEC)
+     # TODO: how to do configure vvdec cleanly?
+     find_package(Threads REQUIRED)
+-    find_package(vvdec 2.3.0)
++    find_package(vvdec 3.0.0)
+     if (vvdec_FOUND)
+         set(vvdec_LIBRARIES vvdec::vvdec)
+     endif()
+@@ -456,7 +456,7 @@ if (WITH_HEADER_COMPRESSION OR WITH_UNCOMPRESSED_CODEC)
+         message("zlib not found")
+     endif()
+ 
+-    find_package(Brotli)
++    # find_package(Brotli)
```

Comment:
Is this because of the whole brotlicommon, brotlienc, and brotlidec linking order issues?

Reply:
Yes...
Shall I add comment about that?
Or can you help to figure out a solution?❤